### PR TITLE
Pr fix chen

### PR DESCRIPTION
**Pull request by chenshoo**

### CODE REVIEW

File: src/handlers/data-version-handler.ts [hunk 1]
```diff
@@ -1,45 +1,55 @@
...
-        // @ts-ignore
-        this.logger = manager.logger;
     }
 
-    async updateDataVersion<Entity>() {
-        let context = this.manager.queryRunner ? this.manager.queryRunner.data ? this.manager.queryRunner.data.context : {} : {};
-        this.logger.debug('Started data version job when inserting/updating entity', {context: context});
-        let result = await this.manager.findOne(DataVersion);
+    public async updateDataVersion<Entity>() {
+        const context = this.manager.queryRunner ? this.manager.queryRunner.data.context : {};
```
Comment (DoctorVoid) [inline src/handlers/data-version-handler.ts]: what if data undefined

File: package.json [hunk 2]
```diff
@@ -67,18 +71,25 @@
...
-    "@types/mocha": "^5.2.7",
+    "@types/jest": "^24.0.11",
     "@types/node": "^12.7.4",
-    "chai": "^4.2.0",
     "husky": "^3.0.9",
-    "mocha": "^6.2.0",
+    "jest": "24.1.0",
     "path": "^0.12.7",
     "pg": "^7.12.1",
+    "pre-commit": "^1.2.2",
```
Comment (osher-sade) [inline package.json]: Remove dependency

File: package.json [hunk 3]
```diff
@@ -67,18 +71,25 @@
...
     "travis-deploy-once": "^5.0.11",
+    "ts-jest": "^24.0.2",
     "ts-node": "^8.3.0",
     "ts-node-register": "^1.0.0",
+    "tslint": "^5.15.0",
+    "tslint-config-prettier": "^1.18.0",
+    "tslint-consistent-codestyle": "^1.15.1",
+    "tslint-eslint-rules": "^5.4.0",
+    "tslint-plugin-prettier": "^2.0.1",
+    "typedoc": "^0.15.0",
```
Comment (osher-sade) [inline package.json]: Remove dependency

**Pull request merged**